Support for Wasm?

I'd really like to be able to run my Web Assembly tests and get coverage reports for them.

First issue, can I specify a custom binary to run e.g. instead of `cargo test`, my test command is `wasm-pack test --node`.

Second issue, I get a compile error when trying to build on `wasm32-unknown-unknown`:

```bash
$ cargo llvm-cov --target wasm32-unknown-unknown
info: when --target option is used, coverage for proc-macro and build script will not be displayed because cargo does not pass RUSTFLAGS to them
   Compiling cfg-if v1.0.0
   Compiling scoped-tls v1.0.0
   Compiling ag-grid-derive v0.1.0
error[E0463]: can't find crate for `profiler_builtins`
  |
  = note: the compiler may have been built without the profiler runtime

For more information about this error, try `rustc --explain E0463`.
error: could not compile `cfg-if` due to previous error
warning: build failed, waiting for other jobs to finish...
error: could not compile `scoped-tls` due to previous error
```

Is the above possible?

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -12,6 +12,8 @@ Note: In this file, do not use the hard wrap in the middle of a sentence for com
 
 ## [Unreleased]
 
+- Update documentation to mention the way to get coverage for `wasm32-unknown-unknown` target.
+
 - Exclude files named `tests.rs`/`*_tests.rs`/`*-tests.rs` from the report by default.
 
 ## [0.6.21] - 2025-10-10

--- a/README.md
+++ b/README.md
@@ -24,6 +24,7 @@ This is a wrapper around rustc [`-C instrument-coverage`][instrument-coverage] a
   - [Get coverage of AFL fuzzers](#get-coverage-of-afl-fuzzers)
   - [Exclude file from coverage](#exclude-file-from-coverage)
   - [Exclude code from coverage](#exclude-code-from-coverage)
+  - [WASM Support](#wasm-support)
   - [Continuous Integration](#continuous-integration)
     - [GitHub Actions and Codecov](#github-actions-and-codecov)
     - [GitLab CI](#gitlab-ci)
@@ -576,6 +577,10 @@ mod tests {
 ```
 
 cargo-llvm-cov excludes code contained in the directory named `tests` and file named `tests.rs`/`*_tests.rs`/`*-tests.rs` from the report by default, so you can also use it instead of `#[coverage(off)]` attribute.
+
+### WASM Support
+
+See the [`wasm-bindgen` guide](https://wasm-bindgen.github.io/wasm-bindgen/wasm-bindgen-test/coverage.html) for the way to get coverage for `wasm32-unknown-unknown` target.
 
 ### Continuous Integration
 

